### DISCUSSION TITLE
show operation

### DISCUSSION THREAD

**Discussion by AndresGarota** [Laravel-Backpack/community-forum#527] [Q&A (Help)] [answered]
i have a relationship field type with subfields,
when i open my entity with edit operation all is ok, but when i open in show operation the field is empty
why?
for best explain this problem i attach my screen

 
![relationship_show_field](https://github.com/Laravel-Backpack/community-forum/assets/132877894/30f49afe-a828-4c3f-8f85-e513b55fbb52)

**Comment by pxpm** [comment, marked as answer]
Hello @AndresGarota there must be some confusion here, `edit/create` operations use `fields`, `show/list` operations use columns. 

In v6 we added an equivalent for each column/field so for the `text` field there is the `text` column as for `relationship` field there is the `relationship` column too. 

v6 is currently in beta, you can follow the docs here: https://backpackforlaravel.com/docs/6.x

I just assume you are not using v6 because the relationship on show operation does not look like that. 

Cheers

**Reply by AndresGarota** [reply]
Hello pxpm, i use v5 version. If I try to eliminate the subfield and I open the entity in show operation, the fields are filled in correctly. The problem is the subfield which does not allow to see the field filled in show operation but it's ok in edit operation

**Reply by o15a3d4l11s2** [reply]
I am also not sure why are there `fields` inside the `show` operation. If you only always need to be able to show and edit the records at the same time, you could just remove the `Preview` operation and only leave `Edit`. In case you also need to be able to `Preview`, then you need to use the https://backpackforlaravel.com/docs/5.x/crud-columns#relationship-pro column, but I think it will not show subfields as you would expect.
